instructions/fault-injection/fault-injection.adoc

```diff
@@ -0,0 +1,150 @@
+= Chaos Testing
+include::_attributes.adoc[]
+
+Apply some chaos engineering by throwing in some HTTP errors or network delays. Understanding failure scenarios is a critical aspect of microservices architecture (aka distributed computing)
+
+[IMPORTANT]
+.Before Start
+====
+You should have only the following virtualservices and destinationrules in the `istio-tutorial` namespace
+
+[source, bash,subs="+macros,+attributes"]
+----
+$ oc -n istio-tutorial get destinationrule
+No resources found.
+
+$ oc -n istio-tutorial get virtualservice
+NAME       GATEWAYS             HOSTS   AGE
+customer   [customer-gateway]   [*]     18h
+----
+
+====
+
+[#503error]
+== HTTP Error 503
+
+By default, recommendation v1, v2 and v3 are being randomly load-balanced as that is the default behavior in Kubernetes/OpenShift
+
+[source,bash,subs="+macros,+attributes"]
+----
+$ oc -n istio-tutorial get pods -l app=recommendation
+NAME                                 READY   STATUS    RESTARTS   AGE
+recommendation-v1-7fbb8f794-ngw58    2/2     Running   0          21h
+recommendation-v2-77bc7bb94d-8wjxz   2/2     Running   0          17h
+recommendation-v3-5786cd744d-96q79   2/2     Running   0          17h
+
+$ while :; do curl http://${INGRESS_GATEWAY} ; done
```

Comment:
We should probably move this line into a script

Reply:
Will do